app/admin/students.rb

```diff
@@ -2,17 +2,80 @@
 
   # See permitted parameters documentation:
   # https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
-  #
-  # Uncomment all parameters which should be permitted for assignment
-  #
-  # permit_params :name, :description, :email, :phone, :mobile_phone, :address, :gender, :birth_date, :website, :twitter, :instagram, :facebook, :observation, :youtube_video, :status
-  #
-  # or
-  #
-  # permit_params do
-  #   permitted = [:name, :description, :email, :phone, :mobile_phone, :address, :gender, :birth_date, :website, :twitter, :instagram, :facebook, :observation, :youtube_video, :status]
-  #   permitted << :other if params[:action] == 'create' && current_user.admin?
-  #   permitted
-  # end
-  
+
+  permit_params :name, :description, :email, :phone, :mobile_phone, :address, :gender, :birth_date, :website, :twitter, :instagram, :facebook, :observation, :youtube_video, :status, :avatar
```

Comment:
Talvez uma quebra de linha ajude aqui
```suggestion
  permit_params :name, :description, :email, :phone, :mobile_phone, :address, 
    :gender, :birth_date, :website, :twitter, :instagram, :facebook, :observation, :youtube_video, :status, :avatar
```